docs/reference/content/getting-started/index.md

```diff
@@ -13,3 +13,5 @@ To help you get started quickly on the new driver, follow:
   * [Installation]({{< ref "getting-started/installation-guide.md" >}})
   * [Quick Tour]({{< ref "getting-started/quick-tour.md" >}})
   * [Admin Quick Tour]({{< ref "getting-started/quick-tour-admin.md" >}})
+  * [Async Quick Tour]({{< ref "getting-started/async-quick-tour.md" >}})
```

Comment:
For now, let's make this two separate lists, and in between a small paragraph like:

  To help you get started with the new async driver, follow: